### ISSUE TITLE
Subscribe to multiple pubs with the same function

### ISSUE THREAD

**Issue by iangilman** [mikehostetler/amplify#46]
Sometimes I'd like to handle a number of similar topics with the same function. I'm imagining something like this: 

```
amplify.subscribe(["foo", "bar", "baz"], function(data, topic) {
    alert("got a " + topic + " with " + data); 
});
```

For that matter, a handy feature for debugging and logging would be subscribeAll().

**Comment by ifandelse** [comment]
amplify.subscribe() can take a space-delimited list of topics currently, and will subscribe the callback for each topic.  For example: 

```
amplify.subscribe("foo bar baz", function(data) {
    alert("got some data " + data); 
});
```

**Comment by iangilman** [comment]
Perfect! Looks like it's a documentation issue then; thanks for adding the label.

**Comment by iangilman** [comment]
oh, wait... can you find out which topic you're receiving? Without that, it's a good deal less useful.

**Comment by ifandelse** [comment]
You probably noticed I revised my reply's code sample, since I, too, thought the callback args included topic - sorry about that.  I'm chatting with the appendTo team to see about opening an issue for adding topic to the callback args....

**Comment by iangilman** [comment]
Awesome, thanks!

**Comment by mgiulio** [comment]
Yes, maybe the documentation should mention this. My code wasn't working because I assumed that I could use event names with spaces.

**Comment by ifandelse** [comment]
@mgiulio - I have a pull request pending that will update the documentation once it's merged.

**Comment by ifandelse** [comment]
@iangilman The pull request I've opened will update the documentation (the amplify.core README) with examples on how you could capture topic inside the subscription callback.

**Comment by iangilman** [comment]
@ifandelse Excellent!
